lib/process-services-cloud/src/lib/task/task-filters/components/base-task-filters-cloud.component.ts

```diff
@@ -45,10 +45,6 @@ export abstract class BaseTaskFiltersCloudComponent implements OnDestroy {
     @Output()
     error: EventEmitter<any> = new EventEmitter<any>();
 
-    /** Emitted when filter is updated. */
-    @Output()
```

Comment:
Why removing this from here? this is in the base component that is being extended from the cloud one